src/main/java/com/happeningnow/Model/Artist.java

```diff
@@ -0,0 +1,25 @@
+package com.happeningnow.Model;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.annotations.UuidGenerator;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+
+@Entity
+@Table(name = "artist")
+public class Artist {
+    @Id
+    @UuidGenerator
+    private UUID id; // one to many
```

Comment:
no need to comment the properties, remove them.

Reply:
ok